vsintegration/src/FSharp.Editor/QuickInfo/QuickInfoProvider.fs

```diff
@@ -33,6 +35,17 @@ type internal FSharpQuickInfoProvider
 
     let xmlMemberIndexService = serviceProvider.GetService(typeof<SVsXMLMemberIndexService>) :?> IVsXMLMemberIndexService
     let documentationBuilder = XmlDocumentation.CreateDocumentationBuilder(xmlMemberIndexService, serviceProvider.DTE)
+    let navigateTo (workspace: Workspace) (range: range)  = async {
+        let documentNavigationService =workspace.Services.GetService<IDocumentNavigationService>()
+        let filePath = System.IO.Path.GetFullPathSafe range.FileName
+        match workspace.CurrentSolution.GetDocumentIdsWithFilePath(filePath) |> List.ofSeq with
+        | id :: _ ->
```

Comment:
See in my PR here https://github.com/Microsoft/visualfsharp/pull/2552/files#diff-ddd3e07525bd2cd12359212293648353R74